### ISSUE TITLE
Enhance the contrast of the focus indicator

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#145]
### Problem

Not only text but also user interface elements and states require sufficient contrast. The minimum contrastratio of 3:1 ensures low vision user can identify and use all interactive elements. This includes focus styles for keyboard users.

The current custom focus styles score below the required minimum contrastratio of 3:1. [More details in this codepen](https://codepen.io/five-oaks/pen/rNLKgEo?editors=1100).

### Solutions

1. Revert to the browser default:  
Remove all custom outlines and allow the browser default to be shown by removing `outline: none` on links and buttons. This would be a big  improvement in Chrome and Edge because they have a very clear default outline. The default outline in Firefox is not so clear unfortunetaly (dotted outline). As long as you do not remove the default outline of the browser, success criterium 1.4.11 Non-text Contrast is met.
2. Implement one custom style for all elements:  
This is a best practice because it's more consistent and therefore easier to follow then mutiple styles in different colors. It's recommended to use a [two-color focus indicator](https://www.w3.org/WAI/WCAG21/Techniques/css/C40) to ensure sufficient contrast with light and dark backgrounds. The default Chromium outline is an example of this.
3. Enhance the current contrast of all custom styles:  
This requires more effort and would be less userfriendly then one consistent style for all elements that can receive focus. But it would be sufficient to comply with 1.4.11 Non-text Contrast.

Possible enhancement: an other way to help keyboard users is to add an animation between the elements that receive focus. [Details and example of focus transition](https://n12v.com/focus-transition/). This is not a requirement to comply with WCAG.

### Links and articles

* [Understanding Success Criterion 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)
* [Codepen: contrast ratio's of outlines](https://codepen.io/five-oaks/pen/rNLKgEo?editors=1100)
* [Creating a two-color focus indicator to ensure sufficient contrast with all components](https://www.w3.org/WAI/WCAG21/Techniques/css/C40)
* [Details and example of focus transition](https://n12v.com/focus-transition/)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)
